### DISCUSSION TITLE
What are the advantages of this compared to the web?

### DISCUSSION THREAD

**Discussion by Daydaylight** [tw93/Pake#501] [Q&A]
It looks like this is a very popular program, but I don't know for a moment what the advantages of this and the web page are, so I hope I can get your answer.

**Comment by Tlntin** [comment]
You can get a customized webpage by injecting js, and then package it and send it to other people who don't understand technology.
By the way, many people prefer applications to web pages, because applications are more convenient to find his startup location.

**Reply by Daydaylight** [reply]
Thanks to your reply, I have a better understanding of the value of this program.

**Comment by tw93** [comment]
Maybe you can try this Google search.
https://www.google.com/search?q=how+to+make+web+to+mac+app&oq=how+to+make+web+to+mac+app&aqs=edge..69i57.14184j0j1&sourceid=chrome&ie=UTF-8#ip=1

**Reply by Daydaylight** [reply]
Thanks for the reply, I understand now.
